libs/net/source/net/socket_server.cpp

```diff
@@ -68,8 +68,20 @@ namespace wolv::net {
         #endif
     }
 
-    void SocketServer::handleClient(SocketHandle clientSocket, const std::atomic<bool> &shouldStop, const Callback &callback) const {
-        std::vector<u8> buffer(this->m_bufferSize);
+    void SocketServer::send(SocketHandle socket, const std::vector<u8> &data) const {
+        ::send(socket, reinterpret_cast<const char*>(data.data()), data.size(), 0);
+    }
+
+    void SocketServer::send(SocketHandle socket, const std::string &data) const {
+        ::send(socket, data.c_str(), data.size(), 0);
+    }
+
+    void SocketServer::close(wolv::net::SocketHandle socket) const {
```

Comment:
Why is this function needed?

Reply:
Forcibly closing a keep alive client, i added this because i thought closeSocket was private